src/components/buttonPanel.jsx

```diff
@@ -0,0 +1,39 @@
+import Button from './button';
+
+function ButtonPanel() {
+  return (
+    <div>
+      <div className="group-1">
+        <Button name="AC" />
+        <Button name="+/-" />
+        <Button name="%" />
+        <Button name="÷" />
+      </div>
+      <div className="group-2">
+        <Button name="2" />
```

Comment:
- [ ] The number 2 is repeated, in here, the number 7 should be used instead to comply with the project requirements, please fix it 👍